usecase/test/test_reply.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+from .. import Reply
+
+class TestReply(unittest.TestCase):
```

Comment:
@fionnfuchs @JannisTriesToCode @timozuther @Ovakefali13 

The `link`, `dict`, `list` and `table` properties of a `Reply` object are transformed into HTML to be displayed nicely in the frontend. Consider if this makes sense in your usecases. If you need more flexibility simply you the `message` field and provide your own HTML in it.

Sneak peek using `link` and `dict`:

<img width="594" alt="image" src="https://user-images.githubusercontent.com/32207204/77614215-ecc81080-6f2c-11ea-9f33-9356db7159ad.png">